docs/faq.md

```diff
@@ -11,32 +11,30 @@ For value transactions, there is the wallet.rs library which provides a stateful
 
 Additionally, there is also a native C ([iota.c](https://github.com/iotaledger/iota.c)) and an alternative, native javascript ([iota.js](https://github.com/iotaledger/iota.js/tree/chrysalis)) client library. 
 
-## What is Dust protection and how does it work?
+## What is Dust Protection and How Does it Work?
 
-Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
+Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this, we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
 
 You can read more about Dust Protection [in the RFC here](https://github.com/iotaledger/protocol-rfcs/pull/32).
 
-## What's happening with Coordicide?
+## What is Happening with Coordicide?
 
 This release called Chrysalis still depends on the Coordinator/Compass to run and issue milestones. The next big release after Chrysalis will be Coordicide where we get rid of that dependency. This release is currently in a research/testing phase and will be the main priority after the Chrysalis release.
 
-## When is a transaction on the network considered final/irreversible?
+## When is a Transaction on the Network Considered Final/Irreversible?
 
 Confirmation times on the new network are on average around 10 seconds. Once a transaction is set to confirm it’s final transaction, you don’t have block confirmations like with blockchain.
```

Comment:
- I'd change the first sentence to:
'On average, confirmation times on the new network are around 10 seconds. '
- Perhaps it's just me as I'm kind of new to this, but I don't understand the second sentence.

Reply:
The "confirmation" is a number of times another block or transaction is placed chronologically after the transaction's block. So its not "block like a goalie" so much as "lego blocks (bricks but you get the idea) stacking after your transaction. So _your_ block is only necessary to confirm, there is no waiting for _other_ blocks. Sorry if that wasn't explained clearly.